src/App.tsx

```diff
@@ -1,9 +1,69 @@
+import React, { useEffect, useState } from 'react';
+
 import './App.scss';
+import { NavLink, Route, Routes } from 'react-router-dom';
+import { Person, Status } from './type/type';
+import { PeopleTable } from './components/PeopleTable';
+import { getPeople } from './api/api';
+import 'bulma';
+
+const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const getLinkStyle = (status: Status): React.CSSProperties => {
+    return status.isActive ? { backgroundColor: '#ccc' } : {};
+  };
+
+  useEffect(() => {
+    getPeople().then(person => setPeople(person));
+  }, []);
```

Comment:
Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened, so move this state and fetching logic to `PeopleTable`